src/index.html

```diff
@@ -1,26 +1,40 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
   <meta charset="utf-8">
   <meta name="viewport" content="width=device-width, initial-scale=1" />
   <title>Knockout TypeScript</title>
-
-  <link
-      rel="stylesheet"
-      href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css"
-      integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm"
-      crossorigin="anonymous">
+  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css"
+    integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm" crossorigin="anonymous">
 
 </head>
-<body>
-<nav class="navbar navbar-dark bg-dark">
-  <div class="container-fluid">
-    <a class="navbar-brand" href="#">
-      <img src="assets/images/image2vector.svg" alt="" width="30" height="24" class="d-inline-block align-text-top">
-      CRM
-    </a>
-  </div>
-</nav>
 
+<body id="app">
+  <nav class="navbar navbar-dark bg-dark">
```

Comment:
@DianaVargasOrellana, Do you think it is a good idea to have the navbar as a different component?, in that way, when need to change any concern about navigation, I should not touch the main file on my application.

Reply:
I have created a new component for the navbar